onadata/libs/tests/utils/test_csv_builder.py

```diff
@@ -629,26 +630,11 @@ def test_csv_export_with_labels(self):
             '_review_status', '_review_comment', '_review_date'
         ]
         self.assertEqual(expected_header, header)
-        labels = next(csv_reader)
-        self.assertEqual(len(labels), 17 + len(csv_df_builder.extra_columns))
-        expected_labels = [
-            'Name', 'age', 'Do you have kids?', 'Kids Name', 'Kids Age',
-            'Kids Name', 'Kids Age', '5. Record your GPS coordinates.',
-            '_gps_latitude', '_gps_longitude', '_gps_altitude',
-            '_gps_precision', 'web_browsers/Mozilla Firefox',
-            'web_browsers/Google Chrome', 'web_browsers/Internet Explorer',
-            'web_browsers/Safari', 'instanceID', '_id', '_uuid',
-            '_submission_time', '_date_modified', '_tags', '_notes',
-            '_version', '_duration', '_submitted_by', '_total_media',
-            '_media_count', '_media_all_received', '_review_status',
-            '_review_comment', '_review_date'
-        ]
-        self.assertEqual(expected_labels, labels)
         rows = []
         for row in csv_reader:
             rows.append(row)
-        self.assertEqual(len(rows), 7)
-        self.assertEqual(rows[4][5], NA_REP)
+        self.assertEqual(len(rows), 8)
+        self.assertEqual(rows[4][16], NA_REP)
```

Comment:
Doesn't seem like the fixture changed... Is there a reason for the additional row ? And Columns ?

Reply:
From the PR description it doesn't seem like this functionality should have changed... Kindly update the PR with all the changes that are going in if this is an expected effect of adding the additional columns for exports with no data

Reply:
There is no fixture file for the test. The contents in the temp csv file are generated from the export function. Those contents have been updated. That is the cause of the new columns in the row data

Reply:
What new columns were added? Seems there are some fixture instances we submit for the export [here](https://github.com/onaio/onadata/pull/2137/files#diff-71defac856691f0cf9680a9c332085b151b84e2222508c82338a6fde859bb98eL556)

_Still a bit confused about what columns have changed... Thought this was only adding functionality to generate export files with headers when no data has been submitted_

Reply:
Ah, I see where the confusion was coming in. Could be something that was updated on the test while testing.
The test doesnt seem to require this changes though at the moment. Thanks for that.